commands/tests/test_upgrade_paths.py

```diff
@@ -0,0 +1,54 @@
+import mock
+import pytest
+
+from leapp.cli.commands.upgrade import upgrade_path
+from leapp.exceptions import CommandError
+
+
+@mock.patch('leapp.cli.commands.upgrade.upgrade_path.get_upgrade_paths_config',
+            return_value={'default': {'7.9': ['8.4'], '8.6': ['9.0'], '7': ['8.4'], '8': ['9.0']}})
+def test_get_target_version(mock_open, monkeypatch):
+
+    monkeypatch.setattr(upgrade_path, 'get_os_release_version_id', lambda x: '8.6')
+    assert upgrade_path.get_target_version('default') == '9.0'
+
+    monkeypatch.setenv('LEAPP_DEVEL_TARGET_RELEASE', '')
+    monkeypatch.setattr(upgrade_path, 'get_os_release_version_id', lambda x: '8.6')
+    assert upgrade_path.get_target_version('default') == '9.0'
+
+    monkeypatch.setenv('LEAPP_DEVEL_TARGET_RELEASE', '1.2.3')
+    assert upgrade_path.get_target_version('default') == '9.0'
+
+    monkeypatch.delenv('LEAPP_DEVEL_TARGET_RELEASE', raising=True)
+    # unsupported path
+    monkeypatch.setattr(upgrade_path, 'get_os_release_version_id', lambda x: '8.5')
+    assert upgrade_path.get_target_version('default') == '9.0'
+
+
```

Comment:
qw ahouls have a test for different flavour as well. at least a simple one, so it's visible that not everything works only for "default".

Reply:
I'd say it's visible for the upgrade_paths_map but ok, I'll add another flavor and a test.
Oh and if it were up to me I'd rename 'flavour' to 'flavor', but there are plenty of occurrences already.